cmd/ak/cmd/server/migrate.go

```diff
@@ -17,12 +16,7 @@ var migrateCmd = common.StandardCommand(&cobra.Command{
 	Args:    cobra.NoArgs,
 
 	RunE: func(cmd *cobra.Command, args []string) error {
-		m, err := svc.ParseMode(mode)
-		if err != nil {
-			return fmt.Errorf("mode: %w", err)
-		}
-
-		db, err := InitDB(common.Config(), m)
+		db, err := InitDB(common.Config(), common.Mode())
```

Comment:
I added the mode to initDB since it didn't work before
now that you change so every command can work in dev mode, maybe you can remove the mode from the signature

Reply:
you can't :( InitDB needs to creates a service first, so it needs the mode.